package.json

```diff
@@ -9,8 +9,7 @@
   "scripts": {
     "dev": "ts-node index.ts",
     "lint": "eslint . --ext .ts,.js",
-    "lint-fix": "eslint . --ext .ts,.js --fix && prettier --write **/*.ts **/*.js",
-    "prod": "tsc -p . && node bin/index.js"
```

Comment:
This prod script is how we compile the program

Reply:
Yeah not absolutely necessary right now since we have the `dev` script but I would add it back, we do need a separate `prod` script. Either using `tsc`, or `ts-node` with `--transpileOnly` flag, it's not recommended to use `ts-node` as-is in prod
https://github.com/TypeStrong/ts-node/issues/104

Reply:
My bad, missed this while merging package.json! Been restored now.